lib/serial_terminal.pm

```diff
@@ -329,6 +348,7 @@ use root-ssh console directly.
 
 sub select_serial_terminal {
     my $root = shift // 1;
+    my $prompt = shift // $root ? '# ' : '> ';
```

Comment:
This should be put to some separate function and reused in lib/susedistribution.pm.

Reply:
If `shift` is a truthy value it'll be `#`, otherwise `>`. If `shift` is `undef` it depends on `$root` instead. Is that what you wanted?

Or did you actually want to assign `shift` with a fallback depending on `$root`? Like this:
```suggestion
    my $prompt = shift // ($root ? '# ' : '> ');
```
Regardless what the answer is, please add braces to clarify the intend (not everyone is totally aware of the operator precedence).